src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      name="bigForm"
+      action="http://localhost:3000/api"
+      method="get"
+      class="bigForm"
+      data-qa="big"
+    >
+
+      <img
```

Comment:
```suggestion
    >
      <img
```
you don't need empty lines between parent and a child

Reply:
пофіксив, дякую